src/main/java/com/agilethought/atsceapi/model/History.java

```diff
@@ -0,0 +1,28 @@
+package com.agilethought.atsceapi.model;
+
+import com.agilethought.atsceapi.domain.FixedExpenses;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import org.springframework.data.annotation.Id;
+import org.springframework.data.mongodb.core.mapping.Document;
+
+@Data
+@AllArgsConstructor
+@Document(collection = "histories")
+public class History {
+    @Id
+    private String id;
+    private byte type;
+    private String user_id;
+    private FixedExpenses fixedExpenses;
+    private int totalHours;
+    private int totalDays;
+    private double costDay;
+    private double costHour;
+    private double taxIVA;
+    private double taxISR_r;
+    private double taxIVA_r;
+    private double total;
+    private double revenue;
+    private boolean status;
```

Comment:
@ivanflores15 didn't you tell us status will remain an integer in the Model and only the DTO will take it as a boolean?

Reply:
And also your Model is missing the "**project cost"** variable.

Reply:
I understood it would be in the opposite way. We define it as a boolean in the Model for storage efficiency reasons, and we define it as an integer in the DTO to satisfy the data type requirement from the front end.

Reply:
yep, DTO should be `Integer` and model `Boolean`

Reply:
Got it, thanks Ivan.